server/routes/api/challenges.js

```diff
@@ -0,0 +1,44 @@
+const router = require('express').Router();
+const { Challenge } = require('../../db');
+
+/**  /api/challenges **/
+
+// get all challenges
+router.get('/', (req, res, next) => {
+  console.log('get all challenges route');
```

Comment:
let's remove these console.logs

Reply:
Removed in the PR just submitted for Image model